src/DynamoCore/DynamoCore.csproj

```diff
@@ -10,6 +10,7 @@
     <AssemblyName>DynamoCore</AssemblyName>
     <UseVSHostingProcess>false</UseVSHostingProcess>
     <DocumentationFile>$(OutputPath)\DynamoCore.XML</DocumentationFile>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
```

Comment:
this should not need to be true? Whats it for?

Reply:
not intended, will remove